content/v2.0/influxdb-templates/stacks/_index.md

```diff
@@ -39,52 +37,118 @@ Use stacks to save time in the following cases:
+
+Your folder structure might look something like this when you are done:
+
+```sh
+influxdb-assets/
+├── buckets/
+│   ├── telegraf_bucket.yml
+├── redis/
+│   ├── redis_overview_dashboard.yml
+│   ├── redis_label.yml
+│   ├── redis_cpu_check.yml
+│   └── redis_mem_check.yml
+├── mysql/
+│   ├── mysql_assets.yml
+└── README.md
+
+```
+  {{% note %}}
+  When exporting a resource in InfluxDB, InfluxDB will automatically create a `meta.name` for that resource. These names should be unique inside you InfluxDB instance, and a good naming convention can help ensure there are no colisions. Changing the `meta.name` of the InfluxDB resource will cause the stack to orphan the resource with the previous name and create a new resource with the updated name.
+  {{% /note %}}
+
+#### Automate the creation of a stack for each folder
+
+Each folder in your Github repository will likely become its own stack, and you'll want to automate the process to apply your changes to your InfluxDB instance. You can do that using some shell scripts and the Influx CLI. Here is an example shell script that will create a `redis` stack, and automatically apply those changes to your instance.
+
+```sh
+echo "Checking for existing redis stack..."
+REDIS_STACK_ID=$(influx stacks --stack-name redis --json | jq -r '.[0].ID')
+if [ "$REDIS_STACK_ID" == "null" ]; then
+    echo "No stack found. Initializing our stack..."
+    REDIS_STACK_ID=$(influx stacks init -n redis --json | jq -r '.ID')
+fi
+
+# Setting the base path
+BASE_PATH="$(pwd)"
+
+echo "Applying our redis stack..."
+cat $BASE_PATH/redis/*.yml | \
+influx apply --force true --stack-id $REDIS_STACK_ID -q
+```
+
+  {{% note %}}
+  The `--json` flag in the InfluxDB CLI is very useful when scripting against the CLI, as it allows you to grab important information easily using [`jq`](https://stedolan.github.io/jq/manual/v1.6/).
+  {{% /note %}}
+
+You can do this for each of the stacks you are maintaining. Each time you make a change to one of the resources in the stack, you can re-run this script and your changes will be applied to the resources in your InfluxDB instance. There's no need to check for changes before re-applying a stack, since re-applying the same resources doesn't change anything.
+
+For a complete example, check out the blog on [Building a GitOps Workflow with InfluxDB Templates]().
+
+#### Automating deployment using Github Actions or CircleCI
+
+Once you have a script to apply changes being made to your local instance, you can start to automate the deployment to other environments as needed. The InfluxDB CLI has the ability to maintain multiple [configuration profiles]() that makes it easy to issue commands against other InfluxDB instances. To apply the same script to a different InfluxDB instance, just change your active configuration profile using the `influx config set` command. You can also set the desired profile dynamically using the `-c, --active-config` flag.
```

Comment:
Did we discuss/identify the script you have to apply changes to your local instance earlier? If so, perhaps we can explicitly state as a step "Create a script to apply changes to your local instance"?